README.md

```diff
@@ -79,6 +79,80 @@ userStream.batch()
 userStream.stop();
 marketStream.stop();
 ```
+### Batching orders
```

Comment:
I don't think it's the right place to add this documentation. So far the README has been very brief and all the doc went into code. It should also rather be consistent between java and python apis.

Reply:
Yes, I wouldn't say that timer functionality is only a type of batching. We only decided to implement timers this way in first version of API that supports them. We support timers without batching and we don't want to exclude a possibility of implementing creating timers in other ways than with batches.

Reply:
Removed. All relevant parts moved to java docs.